package.json

```diff
@@ -2,10 +2,10 @@
   "name": "@optbot/pytools",
   "version": "0.0.0",
```

Comment:
I was going based on our experience with `restapi` and `ui`, but whatever works. I'm not really too concerned about those versions right now, but we can bump them as many times as we want. Looking back at how we did `restpi` and `ui`, maybe those versions (and this one in the future) should actually be 0.1.0 opposed to 0.0.1. Where the middle digit is the minor version opposed to the last digit which is the patch revision.